cypress/framework/selectors/login.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+const alertContent = '//div[@class="v-alert__content"]';
```

Comment:
https://docs.cypress.io/guides/references/best-practices
@MiroDojkic @kronicker @vladimyr  Should we conform to => https://docs.cypress.io/guides/references/best-practices#Selecting-Elements

Reply:
Me React gave us the answers we need, no need for second thoughts 😄 
https://kentcdodds.com/blog/making-your-ui-tests-resilient-to-change#whats-with-the-data-testid-query

Reply:
On se naslanja na ovu pricu https://testing-library.com/docs/queries/bytestid. Imaju jednu zanimljivu izjavu :)